build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
I think that we must keep this lines. I like to use this command inside the container. `docker exec ...`